site/_docs/reference.md

```diff
@@ -1232,17 +1232,17 @@ Note:
 
 ### Non-scalar types
 
-| Type     | Description                | Example type
-|:-------- |:---------------------------|:---------------
-| ANY      | The union of all types |
+| Type     | Description                | Example type |
```

Comment:
Our markdown style is to not have a `|` after the last column. Can you remove it.

(Intellij does not respect that style, so its edits need to be reverted.)

Also revert the changes in spacing, which become spurious diffs.